src/webhooks/options-automator/options-automator.ts

```diff
@@ -1,21 +1,39 @@
+import { v1 } from '@datadog/datadog-api-client';
 import * as Sentry from '@sentry/node';
 import { KnownBlock } from '@slack/types';
 import { FastifyRequest } from 'fastify';
+import moment from 'moment-timezone';
 
 import { OptionsAutomatorResponse } from '@types';
 
 import { bolt } from '@/api/slack';
 import * as slackblocks from '@/blocks/slackBlocks';
-import { FEED_OPTIONS_AUTOMATOR_CHANNEL_ID } from '@/config';
+import {
+  DATADOG_API_INSTANCE,
+  FEED_OPTIONS_AUTOMATOR_CHANNEL_ID,
+} from '@/config';
 
 export async function handler(
   request: FastifyRequest<{ Body: OptionsAutomatorResponse }>
 ) {
   const { body }: { body: OptionsAutomatorResponse } = request;
   await messageSlack(body);
+  await sendOptionAutomatorUpdatesToDataDog(body);
   return {};
 }
 
+export async function sendOptionAutomatorUpdatesToDataDog(
+  message: OptionsAutomatorResponse
+) {
+  // Not sure how detailed we want the text message to be, but this should be fine initially
+  const params: v1.EventCreateRequest = {
+    title: 'Options Automator Update',
+    text: JSON.stringify(message),
+    dateHappened: moment.now(),
+  };
```

Comment:
Set an eventType? Seems like either `info` or `user_update` would make sense.. or maybe `error` or `success` depending on if the option update succeeded or not?

Reply:
Tags we could add if possible (not sure if we have the raw event data or not):

* `sentry_region` (us, de, disney, etc)
* `option_name` (`span-metrics-extraction-enabled`, `project-abuse-quota.attachment-limit`)
* `option_value`? Not sure if this makes sense
* `option_type` (`boolean`, `float`, etc.)

Reply:
Do we want to send an options event for every option that is changed? Or rather, one event for when a group of options are changed?

Right now, we send a slack message whenever a group of options are changed

Reply:
It seems reasonable to send an event when a group of options are changed, so that may make it difficult to have an `error` or `success` eventType. I'll use `info`. For tags, it seems reasonable to add the region name

Reply:
It also seems reasonable to add the names of the options if that's useful

Reply:
Ideally they could be independent or 1 parent event with children events.

But if that's tricky to implement I'm good with 1 event with multiple options for now (although that will mean we can't tag it with any of the option values)

Reply:
@mwarkentin this is what the current event emitted to dd looks like, I'm going to add the other base tags and stuff, but I think we could break down the events sent. I can add more logic here to see if there's any `errors` or `successes`, or just send every option as one event. 


![image](https://github.com/getsentry/eng-pipes/assets/126607112/d3239a5e-5429-4057-ba5d-0db2d11048e7)